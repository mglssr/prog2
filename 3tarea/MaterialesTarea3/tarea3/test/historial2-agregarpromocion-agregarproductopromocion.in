crearHistorial 01/01/2024
crearPromocion 01/12/2023 01/02/2024 10 1
agregarPromocionHistorial
crearPromocion 15/01/2023 15/02/2024 10 2
agregarPromocionHistorial
crearFecha 01/01/2020
crearProducto 1 Producto1 Autor1 Apellido1
agregarProductoAPromocionHistorial 1
crearFecha 02/02/2020
crearProducto 2 Producto2 Autor2 Apellido2
agregarProductoAPromocionHistorial 1
crearFecha 03/03/2020
crearProducto 3 Producto3 Autor3 Apellido3
agregarProductoAPromocionHistorial 2
crearFecha 04/04/2020
crearProducto 4 Producto4 Autor4 Apellido4
agregarProductoAPromocionHistorial 2
liberarHistorial
Fin
