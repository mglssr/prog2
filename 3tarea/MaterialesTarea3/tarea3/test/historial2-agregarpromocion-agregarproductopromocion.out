Bienvenido al programa principal de MercadoFinger. Por favor ingrese su comando:
1> Historial creado con exito.
2> Promocion creada con exito.
3> La promocion fue agregada al historial con exito.
4> Promocion creada con exito.
5> La promocion fue agregada al historial con exito.
6> Fecha creada en forma exitosa.
7> Producto creado en forma exitosa.
8> El producto fue agregado a la promocion de id 1.
9> Fecha creada en forma exitosa.
10> Producto creado en forma exitosa.
11> El producto fue agregado a la promocion de id 1.
12> Fecha creada en forma exitosa.
13> Producto creado en forma exitosa.
14> El producto fue agregado a la promocion de id 2.
15> Fecha creada en forma exitosa.
16> Producto creado en forma exitosa.
17> El producto fue agregado a la promocion de id 2.
18> Historial liberada con exito.
19> Fin.
